templates/views/header.html

```diff
@@ -8,7 +8,12 @@
             </i>
         </div>
         <div class="f-brand item fitted">
-            <img class="f-logo" alt="Forsta" src="{{static 'images/logo_just_text.svg'}}"/>
+            {{#if orgAttrs.banner}}
```

Comment:
All things being the same, go ahead and put the condition around the src attribute itself so we don't have to grok that the only difference is that one attribute.  whitespace can be folded to make it clean like you have the conditions laid out now.

Reply:
Doesn't seem to be work... Can you provide a working example?

Reply:
```
<div {{#if foo}}
            bar="baz"
        {{else}}
            bar="foo"
        {{/if}}
         shoe="foot"></div>
```